File: C_Programs/whileloop5.c
int main(){
	int result,num;
	int i=1;
	printf("enter the num");
	scanf("%d",&num);
	
	while(i<=10){
		result=num*i;
		printf("\n %d * %d = %d",num,i,result);
		i++;
	}
}

File: java programs/Whileloop5.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

public class Whileloop5 {
    public static void main(String[] args) {
        int fact=1;
        Scanner s=new Scanner(System.in);
        System.out.println("enter the number");
        int num=s.nextInt();
        while(num>0){
            fact=fact*num;
            num--;
        }
        System.out.println(fact);
    }
}